solidity/test/bridge/Bridge.Frauds.test.ts

```diff
@@ -413,6 +421,10 @@ describe("Bridge - Fraud", () => {
           testName: "when wallet state is Unknown",
           walletState: walletState.Unknown,
         },
+        {
+          testName: "when wallet state is Closing",
```

Comment:
We should allow to call `submitFraudChallenge` when the wallet is in `Closing` state.
So we should change the `require` in `submitFraudChallenge` to allow to call the function when the state is `Closing` and add unit tests (probably just require that `submitFraudChallenge` does not revert).

Reply:
Indeed, this is covered in https://github.com/keep-network/tbtc-v2/issues/220. I plan to do it in a follow-up PR to avoid complicating this one that focuses only on introducing the new state.